src/ir/effects.h

```diff
@@ -805,7 +789,7 @@ class EffectAnalyzer {
     if (localsWritten.size() > 0) {
       effects |= SideEffects::WritesLocal;
     }
-    if (mutableGlobalsRead.size() + immutableGlobalsRead.size() > 0) {
+    if (mutableGlobalsRead.size()) {
       effects |= SideEffects::ReadsGlobal;
```

Comment:
How about changing this to `SideEffects::ReadsMutableGlobal` not to be confusing?

Reply:
This part is for the C API, where I think we lean towards not making changes as much as possible, so I'm not sure. I'd rather not change it unless there is a strong reason.

Reply:
Sounds good